Application/EdFi.Ods.Common/Exceptions/DistributedCacheException.cs

```diff
@@ -25,5 +24,13 @@ public class SafeDistributedCacheException : DistributedCacheException
 
         public SafeDistributedCacheException(string message)
             : base(message) { }
+
```

Comment:
I don't see this conde in ODS-6139 commit, was it added as part of the migration to .net 8?